### ISSUE TITLE
apparent bug in multivariate cumulative() models

### ISSUE THREAD

**Issue by davharris** [paul-buerkner/brms#321]
First, I wanted to say that this is a great package.  I think I found a bug in the way multi-response cumulative models are generated, though.

It looks like brms (2.0.1) is naming the intercepts incorrectly for multivariate models with the `cumulative` family.  In the `model` section, I'm getting `temp_Intercept` for all my intercepts, instead of `temp_VARNAME_Intercept`.

**Reproducible R example**:

input: 

```
d = data.frame(
  y1 = 1:10, 
  y2 = 1:10,
  x = 1:10
)

f = bf(cbind(y1, y2) ~ x, family = cumulative()) + set_rescor(FALSE)

make_stancode(f, data = d)
```

output:

```
SYNTAX ERROR, MESSAGE(S) FROM PARSER:

variable "temp_Intercept" does not exist.
  error in 'modele08577426730_filee085171d18ba' at line 93, column 73
  -------------------------------------------------
    91:   if (!prior_only) { 
    92:     for (n in 1:N) { 
    93:       target += ordered_logistic_lpmf(Y_y1[n] | mu_y1[n], temp_Intercept); 
                                                                                ^
    94:     } 
  -------------------------------------------------

Error in stanc(model_code = paste(program, collapse = "\n"), model_name = model_cppname,  : 
  failed to parse Stan model 'filee085171d18ba' due to the above error.
Error during wrapup: 
```

**Generated Stan code**:

In the code below, I believe the two instances of `temp_Intercept` should be `temp_y1_Intercept` and `temp_y2_Intercept`, respectively.

```
// generated with brms 2.0.1
functions { 
  /* cumulative-logit log-PDF for a single response 
   * Args: 
   *   y: response category 
   *   mu: linear predictor 
   *   thres: ordinal thresholds 
   *   disc: discrimination parameter 
   * Returns: 
   *   a scalar to be added to the log posterior 
   */ 
   real cumulative_logit_lpmf(int y, real mu, vector thres, real disc) { 
     int ncat = num_elements(thres) + 1; 
     vector[ncat] p; 
     p[1] = inv_logit(disc * (thres[1] - mu)); 
     for (k in 2:(ncat - 1)) { 
       p[k] = inv_logit(disc * (thres[k] - mu)) - 
              inv_logit(disc * (thres[k - 1] - mu)); 
     } 
     p[ncat] = 1 - inv_logit(disc * (thres[ncat - 1] - mu)); 
     return categorical_lpmf(y | p); 
   } 
  /* cumulative-logit log-PDF for a single response 
   * including category specific effects 
   * Args: 
   *   y: response category 
   *   mu: linear predictor 
   *   mucs: predictor for category specific effects 
   *   thres: ordinal thresholds 
   *   disc: discrimination parameter 
   * Returns: 
   *   a scalar to be added to the log posterior 
   */ 
   real cumulative_logit_cs_lpmf(int y, real mu, row_vector mucs, vector thres, real disc) { 
     int ncat = num_elements(thres) + 1; 
     vector[ncat] p; 
     p[1] = inv_logit(disc * (thres[1] - mucs[1] - mu)); 
     for (k in 2:(ncat - 1)) { 
       p[k] = inv_logit(disc * (thres[k] - mucs[k] - mu)) - 
              inv_logit(disc * (thres[k - 1] - mucs[k - 1] - mu)); 
     } 
     p[ncat] = 1 - inv_logit(disc * (thres[ncat - 1] - mucs[ncat - 1] - mu)); 
     return categorical_lpmf(y | p); 
   } 
} 
data { 
  int<lower=1> N;  // total number of observations 
  int Y_y1[N];  // response variable 
  int<lower=2> ncat_y1;  // number of categories 
  int<lower=1> K_y1;  // number of population-level effects 
  matrix[N, K_y1] X_y1;  // population-level design matrix 
  real<lower=0> disc_y1;  // discrimination parameters 
  int Y_y2[N];  // response variable 
  int<lower=2> ncat_y2;  // number of categories 
  int<lower=1> K_y2;  // number of population-level effects 
  matrix[N, K_y2] X_y2;  // population-level design matrix 
  real<lower=0> disc_y2;  // discrimination parameters 
  int prior_only;  // should the likelihood be ignored? 
} 
transformed data { 
  int Kc_y1 = K_y1 - 1; 
  matrix[N, K_y1 - 1] Xc_y1;  // centered version of X_y1 
  vector[K_y1 - 1] means_X_y1;  // column means of X_y1 before centering 
  int Kc_y2 = K_y2 - 1; 
  matrix[N, K_y2 - 1] Xc_y2;  // centered version of X_y2 
  vector[K_y2 - 1] means_X_y2;  // column means of X_y2 before centering 
  for (i in 2:K_y1) { 
    means_X_y1[i - 1] = mean(X_y1[, i]); 
    Xc_y1[, i - 1] = X_y1[, i] - means_X_y1[i - 1]; 
  } 
  for (i in 2:K_y2) { 
    means_X_y2[i - 1] = mean(X_y2[, i]); 
    Xc_y2[, i - 1] = X_y2[, i] - means_X_y2[i - 1]; 
  } 
} 
parameters { 
  vector[Kc_y1] b_y1;  // population-level effects 
  ordered[ncat_y1-1] temp_y1_Intercept;  // temporary thresholds 
  vector[Kc_y2] b_y2;  // population-level effects 
  ordered[ncat_y2-1] temp_y2_Intercept;  // temporary thresholds 
} 
transformed parameters { 
} 
model { 
  vector[N] mu_y1 = Xc_y1 * b_y1; 
  vector[N] mu_y2 = Xc_y2 * b_y2; 
  // priors including all constants 
  target += student_t_lpdf(temp_y1_Intercept | 3, 0, 10); 
  target += student_t_lpdf(temp_y2_Intercept | 3, 0, 10); 
  // likelihood including all constants 
  if (!prior_only) { 
    for (n in 1:N) { 
      target += ordered_logistic_lpmf(Y_y1[n] | mu_y1[n], temp_Intercept); 
    } 
    for (n in 1:N) { 
      target += ordered_logistic_lpmf(Y_y2[n] | mu_y2[n], temp_Intercept); 
    } 
  } 
} 
generated quantities { 
  // compute actual thresholds 
  vector[ncat_y1 - 1] b_y1_Intercept = temp_y1_Intercept + dot_product(means_X_y1, b_y1); 
  // compute actual thresholds 
  vector[ncat_y2 - 1] b_y2_Intercept = temp_y2_Intercept + dot_product(means_X_y2, b_y2); 
} 
```

**Comment by paul-buerkner** [comment]
Thanks! This is already fixed in the github version of brms (see #314).

**Comment by davharris** [comment]
Thanks! Sorry for the duplicate.
